networks/algorand/rpc.csv

```diff
@@ -0,0 +1,57 @@
+getblock-testnet-dedicated,GetBlock,Dedicated,Resent-State,testnet,Up to $1K,Custom,99.90%,NULL,NULL,FALSE,"[""Algorand Native API"",""Indexer API"",""Algo API""]","[""custom limits"",""unlimited requests"",""unlimited rate limit""]","[""Privacy Protected Relays"",""Transaction Sanitizers"",""Dedicated nodes with enterprise-grade security"",""Multi-region infrastructure""]","[""Real-time performance tracking"",""Algorand Metrics Dashboard""]","[""Global""]",NULL,NULL,NULL,FALSE,"[""[Contact](https://account.getblock.io/sign-up)"",""[Docs](https://docs.getblock.io/?q=algorand)""]",,
+getblock-testnet-pay-as-you-go-archive,GetBlock,Pay-as-you-go,Full Archive,testnet,$49/$199/$499,$0.98/$0.9/$0.83/1M CU,99.90%,NULL,NULL,FALSE,"[""Algorand Native API"",""Indexer API"",""Algo API""]","[""50M/220M/600M CU"",""25/100/200 RPS"",""10/25/50 Access Tokens""]","[""Privacy Protected Relays"",""Transaction Sanitizers"",""Dedicated nodes with enterprise-grade security"",""Multi-region infrastructure""]","[""Real-time performance tracking"",""Algorand Metrics Dashboard""]","[""Global""]",NULL,NULL,NULL,FALSE,"[""[Buy](https://getblock.io/pricing/)"",""[Docs](https://docs.getblock.io/?q=algorand)""]",,
+getblock-testnet-enterprise-archive,GetBlock,Enterprise,Full Archive,testnet,Up to $999,Custom,99.90%,NULL,NULL,FALSE,"[""Algorand Native API"",""Indexer API"",""Algo API""]","[""custom limits""]","[""Privacy Protected Relays"",""Transaction Sanitizers"",""Dedicated nodes with enterprise-grade security"",""Multi-region infrastructure""]","[""Real-time performance tracking"",""Algorand Metrics Dashboard"",""Advanced statistics""]","[""Global""]",NULL,NULL,NULL,FALSE,"[""[Contact](https://account.getblock.io/sign-up)"",""[Docs](https://docs.getblock.io/?q=algorand)""]",,
+getblock-testnet-dedicated-archive,GetBlock,Dedicated,Full Archive,testnet,Up to $1K,Custom,99.90%,NULL,NULL,FALSE,"[""Algorand Native API"",""Indexer API"",""Algo API""]","[""custom limits"",""unlimited requests"",""unlimited rate limit""]","[""Privacy Protected Relays"",""Transaction Sanitizers"",""Dedicated nodes with enterprise-grade security"",""Multi-region infrastructure""]","[""Real-time performance tracking"",""Algorand Metrics Dashboard""]","[""Global""]",NULL,NULL,NULL,FALSE,"[""[Contact](https://account.getblock.io/sign-up)"",""[Docs](https://docs.getblock.io/?q=algorand)""]",,
+nodely-mainnet-free,Nodely,Free,Resent-State,mainnet,$0,,99.95%,NULL,NULL,FALSE,"[""Algod API"",""Indexer API"",""KMD API"",""Smart Contract (ASC1) API""]","[""Up to 1K req/s/project"",""150GB req/mo"",""60 req/s/IP"",""50/s transactions/project"",""18/s transactions/IP""]","[""Secure API keys"",""IPFS GW""]","[""Real time monitoring dashboard""]","[""Global""]",NULL,NULL,NULL,FALSE,"[""[Website](https://nodely.io/)"",""[Docs](https://nodely.io/docs/free/start/)""]",,
```

Comment:
Not sure it is really clear that /s/ is per second here, maybe `Up to 1K req/sec/project` or something?